Assurance Cases.md

```diff
@@ -59,14 +59,18 @@ The final gap that I noticed was with evidence E6. [Fail2Ban]( https://pimylifeu
 
-.red[TODO] Insert Evidence 4
+As you can see in Rebuttal R1, installing malicious add-ons are not the only security risk. With Home Assistant having to communicate with outside repositories when downloading add-ons and updates this adds a risk of a MiTM attack. Home Assistant utilizies HTTPS encryption to try to prevent these attacks. Wireshark logs and update logs could also be implemented to monitor any suspicious activity.
+
+As for Rebuttal R2 with third-party add-ons, Home Assistant community-made third-party add-ons are all publicly published on [GitHub](https://github.com/hassio-addons), explained in sub-claim C2. To prevent any risk from installing any malicious third-party add-ons, Home Assistant warns users about the risk of installing add-ons multiple times shown on their [website](https://www.home-assistant.io/hassio/installing_third_party_addons/#:~:text=Installing%20third-party%20add-ons%20Home%20Assistant%20allows%20anyone%20to,you%20can%20use%20our%20example%20add-on%20repository%20at) and within the application. If the user still enables third-party add-ons such as said in Rebuttal R3, Home Assistant makes sure that it points to the official community add-on repo said in sub-claim C4. Evidence E5 states that Home Assistant provides quality ratings on all third-party add-ons within the community repository with 6 being very secure and 1 meaning that users should not install it unless they are 100% sure that they can trust the source. Evidence E4 supports this sub-claim as well by making sure that users provide a github link that they can trust based on Evidence E5 as well as Evidence E6 which Home Assitant can utilize the [Github code scanner](https://github.blog/2020-09-30-code-scanning-is-now-available/) to find security vulnerabilities within the repositories. Though, if a user provides a unsupported/bad github link stated in Undermine UM1, Sub-claim C5 states that Home Assistant runs all add-ons on seperate self-contained docker containers to prevent any malicious add-on from taking over the system with Evidence E7 and E8 to support this claim from [Home Assistant add-on communication document](https://developers.home-assistant.io/docs/add-ons/communication) and [add-on configuration document](https://developers.home-assistant.io/docs/add-ons/configuration). Sub-claim C6 also states that all add-ons are ran in protection enabled mode with Evidence E9 and E10 to support this claim from the [Add-on security document](https://developers.home-assistant.io/docs/add-ons/security) and the [Add-on configuration document](https://developers.home-assistant.io/docs/add-ons/configuration).
```

Comment:
a unsupported should be `an unsupported`.
Fix that spelling of separate.
Also, close to the end of the paragraph `Sub-claim C6 also states that all add-ons are ran` should be `run`.
I would fix the capitalization of GitHub throughout the paragraph to make it more consistent.

Reply:
yeah fix all instances of GitHub as what Sam said and isn't the GitHub code analyzer automatically ran on all source code in GitHub repos? So HA would not have to implement it because GitHub does it automatically.  Also, elaborate a little on what exactly protection mode enabled does, basically state that addons are installed with no privileges and are unable to try and escalate privileges.